### PR TITLE
Minor proofreading & typos

### PR DESCRIPTION
**Pull request by pbarrio**
I just created all the minor changes into a single commit, but let me know if you want to separate them or remove some of the changes.

### CODE REVIEW

File: pauthabielf64/pauthabielf64.rst [hunk 1]
```diff
@@ -754,13 +754,13 @@ Description:
 
    PageBreak
 
-Reocation Compression
+Relocation Compression
 =====================
```
Comment (smithp35) [inline pauthabielf64/pauthabielf64.rst:758]: You'll need to extend the = to include the extra space (CI caught it).

File: pauthabielf64/pauthabielf64.rst [hunk 2]
```diff
@@ -777,13 +777,15 @@ Static Linking
 ==============
 
 The static linker cannot create signed pointers, just as it cannot run
-constructors for static variables, but the C-runtime that runs before
-main can. The static linker must communicate the details of how to
+constructors for static variables, but the runtime that runs before
```
Comment (smithp35) [inline pauthabielf64/pauthabielf64.rst:780]: How about "but the startup code that runs before the program (``main`` in C/C++) can?

Avoids "runtime that runs" which sounds a bit redundant.

There isn't really a good definition that I can point to. https://en.wikipedia.org/wiki/Entry_point uses the term library startup code and main as the start of the user program.

File: pauthabielf64/pauthabielf64.rst [hunk 3]
```diff
@@ -777,13 +777,15 @@ Static Linking
...
+the user code (e.g. in C/C++, before ``main()``) can. The static
+linker must communicate the details of how to
 create the signed pointers by embedding the information in the ELF
-file. The format of the information is platform ABI as it is a
-contract between the static-linker and the C-runtime. One simple
+file. The format of the information is dependent on the platform ABI,
```
Comment (smithp35) [inline pauthabielf64/pauthabielf64.rst:784]: I have a mild preference for "is defined by" than "is dependent on".

**Review by smithp35 — APPROVED** [review]
I've got a couple of alternative suggestion for a couple of the changes, but otherwise LGTM.

**Comment by pbarrio** [comment]
All comments addressed.

**Comment by stuij** [comment]
lgtm, merged

**Pull request merged**